src/app/components/Item/ItemTableRow.jsx

```diff
@@ -16,12 +16,12 @@ const ItemTableRow = ({ item, bibId, getRecord }) => {
   if (item.requestHold) {
     itemLink = item.available ?
       <Link
-        className="button"
+        className="request-button"
         to={`/hold/request/${bibId}-${item.id}`}
         onClick={(e) => getRecord(e, bibId, item.id)}
         tabIndex="0"
       >Request</Link> :
-      <span className="nypl-item-unavailable">Unavailable</span>;
```

Comment:
This actually won't fully work because this `if` case is only for "ReCAP" items which I roughly made the logic for. It's fine because we are still waiting on better flags to check if a bib i from ReCAP so let's leave as is, but this will probably now not display anything at all except for the the Hold Request button.